spec/requests/api/v3/support/authorization.rb

```diff
@@ -28,27 +28,16 @@
 
 require 'spec_helper'
 
-shared_examples_for 'handling anonymous user' do |type, path|
+shared_examples_for 'handling anonymous user' do
   context 'anonymous user' do
-    let(:get_path) { path % [id] }
-
-    context 'when access for anonymous user is allowed' do
-      before { get get_path }
-
-      it 'should respond with 200' do
-        expect(subject.status).to eq(200)
-      end
-
-      it 'should respond with correct type' do
-        expect(subject.body).to include_json(type.to_json).at_path('_type')
-        expect(subject.body).to be_json_eql(id.to_json).at_path('id')
-      end
```

Comment:
Because they are bogus.
They are not connected to the anonymous user in any way, but to "a user with sufficient permissions", whcih is always tested in the main spec.

What is not in here (and usually in the main spec) is the "user with insufficient permissions". Note that I don't use the word "anonymous user" here, because it just has nothing to do with being anonymous.

However, there is one setting that is related to being anonymous, regardless of other permissions: Receiving a 401 as anonymous user, when the setting `login_required?` is enabled. That is why I left the one remaining spec alive, because I could see its value.

The other specs only duplicate the stuff of the main specs while confusing people by using the term "anonymous user".